### PR TITLE
C3177: Search Retailer by Number

### PR DESCRIPTION
**Pull request by ArielWRojas**

### CODE REVIEW

File: IDT.XGen/Main/UI/Pages/BRO/BossRevolutionOrganizerPage.cs [hunk 1]
```diff
@@ -28,5 +42,24 @@ public bool IsBROPageDisplayed()
...
+        public RetailersPage ClickOnRetailersLink()
+        {
+            try
+            {
+                WebDriverManager.Instance.GetWebDriverWait.Until(ExpectedConditions.ElementExists(Utils.GetByLocator("BossRevolutionOrganizerLocators", "AlertLoadingDisplayNone")));
+                CommonActions.ClickElement(this.LnkRetailers);
+            }
+            catch (TimeoutException e)
+            {
+                Console.WriteLine("Boss Revolution Organizer Page was not loaded in the time expected.", e);
```
Comment (wMercadoJala) [inline IDT.XGen/Main/UI/Pages/BRO/BossRevolutionOrganizerPage.cs]: add a TimeoutException something like this:
throw new TimeoutException (message);

File: IDT.XGen/Main/UI/Pages/BRO/LeftMenu.cs [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            leftMenuOptions.Add("Retailers", this.LnkRetailerManagement);
+
+            // Click on the option
+            try
+            {
+                WebDriverManager.Instance.GetWebDriverWait.Until(ExpectedConditions.ElementExists(Utils.GetByLocator("BossRevolutionOrganizerLocators", "AlertLoadingDisplayNone")));
+            }
+            catch (TimeoutException e)
+            {
+                Console.WriteLine("Left menu was not loaded in the time expected.", e);
```
Comment (wMercadoJala) [inline IDT.XGen/Main/UI/Pages/BRO/LeftMenu.cs]: add a TimeoutException something like this:
throw new TimeoutException (message);

File: IDT.XGen/Main/UI/Pages/BRO/BossRevolutionOrganizerPage.cs (hunk 1 above)
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/BRO/BossRevolutionOrganizerPage.cs]: Ok, I updated it

File: IDT.XGen/Main/UI/Pages/BRO/LeftMenu.cs (hunk 2 above)
Comment (ArielWRojas) [reply IDT.XGen/Main/UI/Pages/BRO/LeftMenu.cs]: Ok, I updated it

**Pull request merged**